studio/localStores/storageExplorer/StorageExplorerStore.js

```diff
@@ -97,17 +96,16 @@ class StorageExplorerStore {
     }
   }
 
-  initStore(projectRef, url, serviceKey) {
+  initStore(projectRef, url, serviceKey, protocol = PROJECT_ENDPOINT_PROTOCOL) {
```

Comment:
wont `PROJECT_ENDPOINT_PROTOCOL` be undefined here?

Reply:
AFAIK `PROJECT_ENDPOINT_PROTOCOL` will be always "http"

Reply:
it is not imported in this file, will lead to an error

Reply:
You're right.
Rethinking at it, since `PROJECT_ENDPOINT_PROTOCOL` is always "http" because it is client-side code, maybe I should revert the commit suggestion?